chromadb/config.py

```diff
@@ -88,6 +88,7 @@ class Settings(BaseSettings):  # type: ignore
     chroma_server_headers: Optional[Dict[str, str]] = None
     chroma_server_http_port: Optional[str] = None
     chroma_server_ssl_enabled: Optional[bool] = False
+    chroma_server_api_default_path: Optional[str] = "/api/v1"
```

Comment:
Why this change?

Reply:
I think this was hard-coded in `chromadb/api/fastapi.py`, so I thought to move it out in settings in case there's a need to bump the API version. Also, gives the opportunity to clients to define which API version they want to use when, should more than one version exist on the server (or even communicating with legacy server with v1 API). wdyt?